### ISSUE TITLE
Intermittent false positive for var in protocol

### ISSUE THREAD

**Issue by paulz** [peripheryapp/periphery#304]
```
Pods/Periphery/periphery version
2.4.3
```

We see intermittent failure (about 50% of tries) when running:
```
Run Pods/Periphery/periphery scan --strict --clean-build
Error: Found 1 issue.
* Inspecting project...
* Indexing...
* Analyzing...
...
.../VoltagePlotUpdater.swift:9:9: warning: Property 'needsUpdate' is assigned, but never used
* Seeing false positives?
...
Error: Process completed with exit code 1.
```

```
public protocol VoltagePlotUpdaterType {
    var needsUpdate: Bool { get set }
    func startUpdating(plot: VoltagePlot)
}
```

used in the same target:
```
public class VoltagePlotUpdater: VoltagePlotUpdaterType {
    public var needsUpdate = false

    private func publishUpdated(plot: VoltagePlot) {
        if needsUpdate {
            plot.publishPoints()
            needsUpdate = false
        }
    }
}
```

**Comment by ileitch** [comment]
I believe this bug is already fixed by #291, however it's not released yet. I hope to cut the release sometime late next week. In the meantime it'd be great if you see if your issue is resolved in master.

**Comment by paulz** [comment]
thank you @ileitch, we want to try it, but unable to do so without the release 

we install `Periphery` via cocoa pods on macOS:
```
pod 'Periphery',              # find unused code
  :git => 'https://github.com/peripheryapp/periphery'
```

First we tried our original command `Pods/Periphery/periphery scan --strict --clean-build`
but seems we are now supposed to use `Pods/Periphery/scripts/scan`

When running `Pods/Periphery/scripts/scan`, we see
```
Pods/Periphery/scripts/scan --strict --clean-build 
warning: '--enable-test-discovery' option is deprecated; tests are automatically discovered on all platforms
error: root manifest not found
```

Then we tried a variation that will build periphery first, 
```
pushd ./Pods/Periphery/scripts/
./scan <Our project folder>
```

but we got this as output:
```
[450/450] Linking periphery

* Build Completed!
./scan: line 3: ./.build/debug/periphery: No such file or directory
```

Hope to try it when release is available

**Comment by ileitch** [comment]
I've just released 2.5.0 which should resolve this issue. Please give it a try and report back if you're still having issues.
